playbook/app/pb_kits/playbook/pb_title/_title.scss

```diff
@@ -17,10 +17,30 @@
   &[class*=_4] {
     @include pb_title_4;
 
-    &[class*=_link]  {
+    &[class*="link"] {
       color: $primary;
     }
```

Comment:
This colors should be moved because right now they're inside the  `[class*=_link]` which means they're available only on h4 tag. Let me know if you need help.